modules/sms_user/config/install/sms_user.settings.yml

```diff
@@ -1,6 +1,15 @@
 active_hours:
   status: false
   ranges: []
-registration_enabled: false
-allow_password: false
-new_account_message: ''
+account_registration:
+  all_unknown_numbers:
+    status: false
+    reply_message: ''
+  formatted:
+    status: false
+    incoming_messages: []
+    activation_email: true
+    reply:
+      status: false
+      message: ''
+      message_failure: ''
```

Comment:
Like I said... https://github.com/dpi/smsframework/pull/21#issuecomment-212504314 naming is TBD. + the config schema is missing label/descriptions.

I dont expect to overload the names with a full description of functionality, im leaning towards 'unknown_numbers'.

'formatted' needs a better name.